views/templates/block/channelInfo.twig

```diff
@@ -1,47 +1,219 @@
         {% endif %}
         {% if upgradeInfo.md5 is not empty %}
-            <div style="clear:both;">
-                <label class="label-small">{{ 'MD5 hash:'|trans }}</label>
-                <span class="md5">{{ upgradeInfo.md5 }}</span>
+            <div class="row">
+                <div class="col-xs-3 col-md-1">
+                    <label class="label-small">{{ 'MD5 hash:'|trans }}</label>
+                </div>
+                <div class="col-xs-9 col-md-11">  
+                    <span class="md5">{{ upgradeInfo.md5 }}</span>
+                </div>
             </div>
         {% endif %}
         {% if upgradeInfo.changelog is not empty %}
-            <div style="clear:both;">
-                <label class="label-small">{{ 'Changelog:'|trans }}</label>
-                <a class="changelog" href="{{ upgradeInfo.changelog }}">{{ 'see changelog'|trans }}</a>
+            <div class="row">
+                <div class="col-xs-3 col-md-1">
+                    <label class="label-small">{{ 'Changelog:'|trans }}</label>
+                </div>
+                <div class="col-xs-9 col-md-11">  
+                    <a class="changelog" href="{{ upgradeInfo.changelog }}">{{ 'see changelog'|trans }}</a>
+                </div>
             </div>
         {% endif %}
+        <div class="row">
+            <div class="col-xs-3 col-md-1">
+                <label class="label-small">{{ 'PHP compatibility:'|trans }}</label>
+            </div>
+            <div class="col-xs-9 col-md-11">
+                <span class="phpcompatibility">{{ 'Your PHP version must be compatible with both your current PrestaShop version and the one you want to upgrade to.
+                    For example, if your current version is 1.6.1.x and you want to upgrade to 1.7.8, only the PHP 7.1 would be compatible with both.'|trans }}
+                </span>
+                <table class="table-striped">
+                <thead>
+                    <tr>
+                    <th style="background:#fff;"></th>
+                    <th colspan="12" style="text-align:center" class="phpversion">PHP Version</th>
+                    </tr>
+                    <tr class="h-version-titles">
+                    <th>PrestaShop Version</th>
+                    <th>≤&nbsp;5.1</th>
+                    <th>5.2</th>
+                    <th>5.3</th>
+                    <th>5.4</th>
+                    <th>5.5</th>
+                    <th>5.6</th>
+                    <th>7.0</th>
+                    <th class="target target-col" style="text-align:center">7.1 <p class="green">Required</p></th>
+                    <th>7.2</th>
+                    <th>7.3</th>
+                    <th>7.4</th>
+                    <th>8.0</th>
+                    <th>≥&nbsp;8.1</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td class="target">1.6.1.x <span class="green">Current</span></td>
```

Comment:
Should be automatically generated, not managed by hands

Reply:
The specs specify that this table is written statically... for now.

The idea was to go fast, and generate this table dynamically in the next version, displaying only relevant pretashop versions and highlighting the right column.